### PR TITLE
Add a standalone FacilityCode bruteforcer for HID

### PR DESCRIPTION
**Pull request by ss23**
This requires a known card number, but often works as 1 is a default.

My first stand-alone module, so let me know if there are any pointers etc :)

### CODE REVIEW

**Comment by iceman1001** [comment]
Nice,  
one question, what is it that you actually log?   There is no way for the proxmark3 to know when there is a successful attempt

**Comment by ss23** [comment]
Since there's no automated way to tell, the only thing that is logged is when a user presses the button manually. The idea being that while doing the bruteforce, you wait for the system to accept a given FC, then you press the button so you log it (or the FC after it), that way you know what it was later.

It's not an ideal situation, but I figure this is more useful than trying to count manually.

File: armsrc/Standalone/lf_hidfcbrute.c [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+#include "lfops.h"
+#include "BigBuf.h"
+#include "fpgaloader.h"
+#include "parity.h"
+
+// What card number should be used for the bruteforce?
+// In some systems, card number 1 is valid, so this may be a good starting point.
+#define CARD_NUMBER 1
+
+#define LF_HIDCOLLECT_LOGFILE "lf_hidlfcollect.log"
```
Comment (iceman1001) [inline armsrc/Standalone/lf_hidfcbrute.c:44]: maybe rename the file to relate to _brute_ ?
like  `lf_hidbrute.log`

**Comment by iceman1001** [comment]
ok,   so it will iterate 256 times regardless  and first button press starts the brute function. 
then every button press logs current number.   

ok,

**Pull request merged**